projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -9,14 +10,19 @@
 from vdk.internal.builtin_plugins.config import vdk_config
 from vdk.internal.builtin_plugins.run.job_context import JobContext
 from vdk.internal.core import errors
+from vdk.internal.core.config import ConfigurationBuilder
 from vdk.internal.core.statestore import CommonStoreKeys
 
+LOGGING_ENDPOINT_URL = "LOGGING_ENDPOINT_URL"
```

Comment:
It's a Syslog endpoint and some generic logging endpoint, Let's be clear and specific.

Reply:
noted, renamed vars